app/build.gradle

```diff
@@ -3,58 +3,54 @@ apply plugin: 'kotlin-android'
 
 android {
     namespace 'com.burhanrashid52.photoediting'
-    compileSdkVersion 31
+    compileSdkVersion 33
 
     defaultConfig {
         applicationId "com.burhanrashid52.photoediting"
         minSdkVersion 14
-        targetSdkVersion 31
+        targetSdkVersion 32
```

Comment:
Couldn't set 33 because of Glide bug https://github.com/bumptech/glide/issues/4940